src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -185,7 +203,7 @@ def build_summary_data(
                 type__in=(ActivityType.SET_REGRESSION.value, ActivityType.SET_ESCALATING.value),
             )
             if regressed_or_escalated_groups_today:
-                for activity in regressed_or_escalated_groups_today:
+                for activity in regressed_or_escalated_groups_today[:4]:
```

Comment:
Making sure we limit this so the message isn't gigantic and we don't hit the message limit for Slack, especially for sentry where we have so many deploys for one commit